packages/blade/src/components/Carousel/_decisions/decisions.md

```diff
@@ -0,0 +1,176 @@
+| shouldAddStartEndMargin  | `boolean`                              | `false`                                  | If true, adds extra margin before and after the first/last slides so that they align in center, this prop can be used to achive carousel which is in [GST page](https://razorpay.com/gst-number-search/pan/) |          |
+| showIndicators           | `boolean`                              | `true`                                   | Toggles the visibility of indicators                                                                                                                                                                         |          |
+| scrollOverlayColor             | `BrandColorTokens, SurfaceColorTokens` | `surface.background.level1.highContrast` | Sets the color of the overlay, so that consumers can blend the overlay with the background color, if this is not set the overlay won't be rendered                                                           |          |
+| navigationButtonPosition | `bottom,side`                          | `bottom`                                 | Sets the position of navigation button                                                                                                                                                                       |          |
+| navigationButtonVariant    | `filled,stroke`                        | `filled`                                 | Styles to be used depending on the emphasis you want to give to the navigation buttons                                                                                                                       |          |
+| indicatorVariant         | `gray,white,blue`                      | `gray`                                   | Variants to be used depending on the background and the emphasis you want to give to the indicator.                                                                                                          |          |
+| onChange                 | `(slideIndex: number) => void`         | `undefined`                              | Callback which gets fired everytime a slide changes, useful for analytic events (only runs on user interaction not on autoPlay)                                                                              |          |
+| accessibilityLabel       | `string`                               | `undefined`                              | Accessibility label for the carousel, this will let screen reader users know what content the carousel holds (eg: "Product carousel")                                                                        |          |
+
+- Additionally we will also expose few styled props, namely margin/width etc so that users can add negative margins to create end to end bleed carousel.
+- To let consumers scroll to any specific slide we will expose a `ref` with `goToSlide` method.
```

Comment:
Should we provide next, previous method ? If we intend to provide only ref then we can provide these methods on ref also.

Reply:
We will actually look into adding controlled state in the carousel, so no need to expose these refs.